keyboards/bt66tech/bluepill/keyboard.c

```diff
@@ -0,0 +1,20 @@
+#include "ch.h"
+#include "hal.h"
+#include "util.h"
+#include "quantum.h"
+
+#ifdef BOARD_STM32_F103_STM32DUINO
```

Comment:
Given `BOARD_STM32_F103_STM32DUINO` will always be true, and it blocks startup by half a second to flash a LED, i'm not sure this code add much value.